x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/ShrinkStep.java

```diff
@@ -62,7 +62,7 @@ public void performAction(IndexMetadata indexMetadata, ClusterState currentState
         resizeRequest.getTargetIndexRequest().settings(relevantTargetSettings);
 
         getClient().admin().indices().resizeIndex(resizeRequest, ActionListener.wrap(response -> {
-            listener.onResponse(response.isAcknowledged());
+            listener.onResponse(true);
```

Comment:
Can you add a comment here about why this is hardcoded to true so we don't accidentally change it back at some point in the future?